tests/functional_small/ops/ops_tpetra_multi_vector.cc

```diff
@@ -4,7 +4,7 @@
 
 TEST_F(tpetraMultiVectorGlobSize15Fixture, multi_vector_clone)
 {
-    auto a = pressio::ops::clone(*myMv_);
+    auto a = ::pressio::ops::clone(*myMv_);
```

Comment:
i was not worried about the namespace in the tests since here this is a single executable with nothing else but this is fine too!